dev-setup/debian-setup.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+# First we should go home
+cd ~/
+
+# Add PostgreSQL's repository to your system:
+echo "Adding PostgreSQL's repository to your system..."
+sudo sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list'
+
+# Import the repository signing key:
+echo "Importing the PostgreSQL's repository signing key..."
+wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
+
+# Update the package lists:
+echo "Updating package lists..."
+sudo apt update 
+
+# Install Rust build tools and dependencies:
+echo "Installing development tools and dependencies..."
+sudo apt install -y build-essential libssl-dev pkg-config gettext gcc g++ curl git
+
+# Install PostgreSQL:
+echo "Installing PostgreSQL..."
+sudo apt install postgresql libpq-dev
+
+# Install Rust:
+echo "Installing Rust..."
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+
+# Install Rust toolchain:
+echo "Installing Rust stable toolchain..."
+rustup install stable
+
+# Install Rust tools:
+echo "Installing Rust tools..."
+rustup component add rustfmt clippy cargo-watch
+cargo install diesel_cli --no-default-features --features "postgres"
+
+# Clone Aardwolf:
+echo "Switching to home directory"
+cd ~/
+echo "Cloning Aardwolf..."
+git clone https://github.com/aardwolf/aardwolf
```

Comment:
question: What is the reason for including this in the setup script?  I would have expected people to be running this *after* cloning it

Reply:
Yeah, that makes sense.  This was probably me auto-translating from the GitHub Action.
it doesn't *really* make sense unless the setup scripts are downloaded from outside of GitHub.